Fitbit.Portable/FitBitHttpClientMessageHandler.cs

```diff
@@ -1,38 +1,59 @@
 ﻿namespace Fitbit.Api.Portable
 {
+    using System;
     using System.Diagnostics;
     using System.Net.Http;
     using System.Threading;
     using System.Threading.Tasks;
 
-    public class FitbitHttpClientMessageHandler : DelegatingHandler
```

Comment:
Renaming in manually can be a pain, the renaming refactoring in VS should do it but I know for a fact that Resharper renaming does files as well. If you don't have resharper I would strongly recommend looking into it :-)